frontend/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
```

Comment:
Update your README  -

Add  brief description about your project.
You can also add a checklist that can be used to track all the topics covered until now.

For reference - https://github.com/kalviumcommunity/openbadgx/blob/main/README.md